src/SourceBuild/content/eng/build.ps1

```diff
@@ -42,9 +61,9 @@ function Build {
 
   MSBuild $buildProj `
     $bl `
-    --tl:off `
     /p:Configuration=$configuration `
```

Comment:
Should this be moved into `$arguments`?

Reply:
I kept it in sync with eng/common/build.ps1 and eng/common/build.sh. They pass Configuration in explicitly as well.